backstop_data/engine_scripts/puppet/home-dashboard/configure-dashboard.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = async (engine, scenario, vp) => {
```

Comment:
if you don't need `scenario` and `vp`, then don't list them

Reply:
I checked at other places to fix this but these 3 params are passed in each function in every file. I guess to maintain a standard we can have these 3 params in the function.